### PR TITLE
Convert op specification refactoring.

### PR DESCRIPTION
**Pull request by jdanieck**
### Details:
 - Convert operation specification refactored

### Tickets:
 - 46095

### CODE REVIEW

File: docs/ops/type/Convert_1.md [hunk 1]
```diff
@@ -18,23 +27,15 @@
 
 **Inputs**
 
-* **1**: A tensor of type T. **Required.**
+* **1**: A tensor of type T and arbitrary shape. **Required.**
```
Comment (ggalieroc) [inline docs/ops/type/Convert_1.md:30]: *T*

File: docs/ops/type/Convert_1.md [hunk 2]
```diff
@@ -4,7 +4,16 @@
 
 **Category**: type conversion
 
-**Short description**: Operation converts all elements of the input tensor to a type specified in the *"destination_type"* attribute.
+**Short description**: Operation converts all elements of the input tensor to a type specified in the *destination_type* attribute.
```
Comment (ggalieroc) [inline docs/ops/type/Convert_1.md:7]: *Convert* operation performs element-wise conversion, on a given input tensor, to a type...

**Review by ggalieroc — COMMENTED** [review]
Maybe use backticks to enclose string type of attribute?

File: docs/ops/type/Convert_1.md [hunk 3]
```diff
@@ -4,7 +4,16 @@
 
 **Category**: type conversion
```
Comment (ggalieroc) [inline docs/ops/type/Convert_1.md:5]: I see in many PRs the category enclosed by *. Up to you, but probably starting by capital letter. eg:
\*Type conversion\*

File: docs/ops/type/Convert_1.md (hunk 1 above)
Comment (jdanieck) [reply docs/ops/type/Convert_1.md]: Done [here](https://github.com/openvinotoolkit/openvino/pull/5530/commits/3921c0f91cf899ed0c2577ddaf0b5841159669c8)

File: docs/ops/type/Convert_1.md (hunk 2 above)
Comment (jdanieck) [reply docs/ops/type/Convert_1.md]: Done [here](https://github.com/openvinotoolkit/openvino/pull/5530/commits/3921c0f91cf899ed0c2577ddaf0b5841159669c8)

File: docs/ops/type/Convert_1.md (hunk 3 above)
Comment (jdanieck) [reply docs/ops/type/Convert_1.md]: Done [here](https://github.com/openvinotoolkit/openvino/pull/5530/commits/f4e8c0dd48f2d34f44b7692c2426a194fa4f0852)

**Review by ggalieroc — APPROVED** [review]

**Review by lazarevevgeny — APPROVED** [review]

**Pull request merged**